samples/nrf9160/multicell_location/src/main.c

```diff
@@ -98,11 +100,24 @@ static void lte_handler(const struct lte_lc_evt *const evt)
 	case LTE_LC_EVT_NEIGHBOR_CELL_MEAS:
 		LOG_INF("Neighbor cell measurements received");
 
-		/* Copy current and neighbor cell information. */
+		if (evt->cells_info.current_cell.id == 0) {
+			LOG_DBG("Cell ID not valid..");
```

Comment:
"Cell ID not valid."  <-- one period

Reply:
weird, the serial viewer i use adds a period 🤦  
i assumed it was the logger that did this.